lib/jsonapi_spec_helpers/helpers.rb

```diff
@@ -56,6 +56,7 @@ def json_ids(integers = false)
 
     def validation_errors
       @validation_errors ||= {}.tap do |errors|
+        return if json['errors'].nil?
```

Comment:
Could we change this like to

```ruby
errors = json['errors'] || []
errors.each do |e|
```

That way the method always returns a hash, and something like `validation_errors[:foo]` won't fail

Reply:
The `errors` variable is already taken as the `self` of `{}.tap`. But yes, I have made that change by using a different variable.

I am more familiar with Javascript than Ruby, so I thought returning would still return—wait. I literally just thought of a better way as I was writing my explanation of what I did...

Reply:
Okay. It is fixed below.

My heart knows Ruby better than Javascript, but my brain knows Javascript better than Ruby.